static/src/javascripts/projects/commercial/modules/mobile-sticky.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import config from 'lib/config';
+import fastdom from 'lib/fastdom-promise';
+import { getCookie } from 'lib/cookies';
+import { addSlot } from 'commercial/modules/dfp/add-slot';
+import { commercialUsMobileSticky } from 'common/modules/experiments/tests/commercial-us-mobile-sticky';
+import { isInVariantSynchronous } from 'common/modules/experiments/ab';
+
+const createAdSlot = (): HTMLDivElement => {
+    const adSlot: HTMLDivElement = document.createElement('div');
+    adSlot.id = 'dfp-ad--mobilesticky';
+    adSlot.className = 'js-ad-slot ad-slot ad-slot--mobile-sticky';
+    adSlot.setAttribute('data-link-name', 'ad slot mobilesticky');
+    adSlot.setAttribute('data-name', 'mobilesticky');
+    adSlot.setAttribute('data-mobile', '320,50');
+    adSlot.setAttribute('aria-hidden', 'true');
+
+    return adSlot;
+};
+
+const createAdWrapper = (): HTMLDivElement => {
+    const wrapper: HTMLDivElement = document.createElement('div');
+    wrapper.className = 'mobilesticky-container';
+    wrapper.appendChild(createAdSlot());
+    return wrapper;
+};
+
+const isInNA = (): boolean =>
+    (getCookie('GU_geo_continent') || 'OTHER').toUpperCase() === 'NA';
+
+export const init = (): Promise<void> => {
+    if (
+        isInNA() &&
```

Comment:
It's not causing any issues but I think it's unnecessary to check `isInNA()` here as we're checking this in the `canShow` of the `commercialUsMobileSticky` test so there's no chance `isInVariantSynchronous(commercialUsMobileSticky, 'variant')` can be true if there not  in the US?

Reply:
This ad slot will only run on the North American continent. It's present here to make sure of that. And it's present again in AB test's canRun function to make sure it doesn't include users from outside the NA in the test.